### PR TITLE
Login page changes

### PR DESCRIPTION
**Pull request by ronhenson**
Removed login on toolbar,
Inserted Sign up on login page,
removed password from console

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[0129/151620.937:ERROR:registration_protocol_win.cc(130)] TransactNamedPipe: The pipe has been ended. (0x6D)
```
Comment (dereklance) [inline debug.log]: What is this file? SHould we be tracking this?

File: src/app/components/login/login.component.scss [hunk 2]
```diff
@@ -13,3 +15,11 @@
   margin-right: 10px;
   padding: 10px;
 }
+
+.signout-button {
```
Comment (dereklance) [inline src/app/components/login/login.component.scss:19]: unused css class

File: src/app/components/toolbar/toolbar.component.ts [hunk 3]
```diff
@@ -30,5 +32,8 @@ export class ToolbarComponent implements AfterContentChecked {
   logoutClick(): void {
```
Comment (dereklance) [inline src/app/components/toolbar/toolbar.component.ts]: method should be in the auth service

File: src/app/components/toolbar/toolbar.component.ts [hunk 4]
```diff
@@ -13,8 +14,9 @@ export class ToolbarComponent implements AfterContentChecked {
...
     private toolbarService: ToolbarService,
     private ref: ChangeDetectorRef,
-    private authService: AuthService
-  ) {
+    private authService: AuthService,
+    private router: Router
+    ) {
     this.toolbarService.getNavChangeEmitter().subscribe((title: string) => this.toolbarTitle = title);
   }
   isLogin = false;
```
Comment (dereklance) [inline src/app/components/toolbar/toolbar.component.ts]: where is this variable being used?

**Review by dereklance — CHANGES_REQUESTED** [review]

File: debug.log (hunk 1 above)
Comment (ronhenson) [reply debug.log]: no.  I'll remove.  It popped up when running npm test

**Review by dereklance — APPROVED** [review]

**Pull request merged**